### ISSUE TITLE
Seemingly useless code in Form

### ISSUE THREAD

**Issue by PhilippGrashoff** [atk4/ui#814]
Inside Form->ajaxSubmit():
```
      $caught = function ($e, $useWindow) {
                $html = '<div class="header"> '.
                        htmlspecialchars(get_class($e)).
                        ' </div> <div class="content"> '.
                        ($e instanceof \atk4\core\Exception ? $e->getHTML() : nl2br(htmlspecialchars($e->getMessage()))).
                        ' </div>';
                $this->app->terminate(json_encode(['success' => false, 'message' => $html, 'useWindow' => $useWindow]));
            };
```

I cannot find any usage of this function, it seems it can be deleted.

**Comment by DarkSide666** [comment]
Yes this function (variable $caught` is not used anywhere so it's safe to remove it or at least comment it.
Maybe that's some idea somebody have written down but didn't use it afterwards :)

**Comment by abbadon1334** [comment]
we speak about on gitter, when i raise the exception to be caught on app i forget to remove it.
As i state din chat we need to understand this parameter : `'useWindow' => $useWindow` because if it is raised in app->caughtException this parameter is missing.

To solve this, we can intercept the Exception type and if is ValidationException add it, but this is my only concern about it, because ui didn't see in other cases the option `useWindow` (@ibelar)

**Comment by ibelar** [comment]
$useWindow = true will have apiService display error in a custom html window. The default value, false, will display error in an html window base on Fomantic-UI modal.

We did this at the time because displaying certain type of error in form did not display well when using Modal, was probably Semantic-UI modal also at the time. Error was not showing complete stack and I Think it was data error but could not remember.

Since @abbadon1334 refactor $exception, we can safely remove that part of the code.

**Comment by acicovic** [comment]
Ok, so can someone who knows the code well remove it so we can close this?
